recipes/indi/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "indi" %}
+{% set version = "1.8.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/indilib/indi/archive/v{{ version }}.tar.gz
+  sha256: 82004db3d1e45f8852c1b871fdb925a9ce0afda4f8f92de0a197b9b868933abf
+
+build:
+  number: 0
+  skip: True  # [win]
```

Comment:
This should probably have a `run_exports` section: https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#export-runtime-requirements

Reply:
What would I put in it? I read the linked doc but it sounds like listing, e.g., cfitsio as a "host" dependency includes its run_exports implicitly

Reply:
So this affects your downstream packages, so they don't have to explicitly have to have host & run deps, but can have the host deps propogate